website/server/models/task.js

```diff
@@ -64,6 +66,11 @@ export let TaskSchema = new Schema({
     winner: String, // user.profile.name of the winner
   },
 
+  group: {
+    id: {type: String, ref: 'Group', validate: [validator.isUUID, 'Invalid uuid.']},
+    broken: {type: String, enum: ['GROUP_DELETED', 'TASK_DELETED', 'UNSUBSCRIBED', 'GROUP_TASK_NOT_FOUND']}, // CHALLENGE_TASK_NOT_FOUND comes from v3 migration
```

Comment:
yeah because we don't don't really use CHALLENGE_TASK_NOT_FOUND but I messed up with the v3 migration so some task got unlinked even though it shouldn't have. But it's not really used in the code, it was just to identify what needed to be fixed from the migration